model/src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -252,7 +252,8 @@ case class MethodRepoConfigurationExport(
 case class WorkspaceListResponse(accessLevel: WorkspaceAccessLevel,
                                  workspace: Workspace,
                                  workspaceSubmissionStats: WorkspaceSubmissionStats,
-                                 owners: Seq[String])
+                                 owners: Seq[String],
+                                 public: Option[Boolean])
```

Comment:
That was the story requirements. It could certainly be added in more places, but those were not in scope.